src/app/page/cabinet/userDecks/user.decks.component.html

```diff
@@ -95,9 +95,13 @@ <h4 class="modal-title">{{dialogName}}</h4>
           </table>
         </div>
         <div class="modal-footer">
-          <button *ngIf="name !== '' && description !== '' && categoryId != -1" class="btn btn-primary btn-pointer"
+          <button *ngIf="name && description && categoryId != -1" class="btn btn-primary btn-pointer"
```

Comment:
-1 is still here...

Reply:
Sorry, I have no idea how to fix that. I have tried a lot of different ways but noone works.
Of course, I can assign that variable undefined or null but in any case it'll be interpreted as a string.